src/sentry/search/events/fields.py

```diff
@@ -2759,6 +2809,39 @@ def _resolve_user_misery_function(self, args: Mapping[str, str], alias: str) ->
             alias,
         )
 
+    def _resolve_percentile(
+        self, args: Mapping[str, str], alias: str, fixed_percentile: float = None
+    ) -> SelectType:
+        measurement_aliases = {
+            MEASUREMENTS_FRAMES_SLOW_RATE,
+            MEASUREMENTS_FRAMES_FROZEN_RATE,
+            MEASUREMENTS_STALL_PERCENTAGE,
+        }
+        column = self.column(args["column"])
+        if args["column"] in measurement_aliases:
+            column = Function(
+                "if",
+                [
+                    Function(
+                        "greater",
+                        [self.column("measurements.frames_total"), 0],
+                    ),
+                    Function(
+                        "divide",
+                        [
+                            self.column("measurements.frames_slow"),
+                            self.column("measurements.frames_total"),
+                        ],
+                    ),
+                    0,
+                ],
+            )
```

Comment:
From a Slack conversation, this block of code doesn't really belong here but rather in an Argument resolver. We need to figure out a nicer way to implement it.

Reply:
Going to move the measurement columns to `field_alias_converter`

Reply:
👍 we've moved columns to `field_alias_converter`!